src/toast/todmap/conviqt.py

```diff
@@ -507,47 +507,39 @@ def exec(self, data):
                 beam_file = self._beam_file[det]
             except TypeError:
                 beam_file = self._beam_file.replace("DETECTOR", det)
-                beam_file_i00 = beam_file.replace(".fits", "_I000.fits")
-                beam_file_0i0 = beam_file.replace(".fits", "_0I00.fits")
-                beam_file_00i = beam_file.replace(".fits", "_00I0.fits")
+                beam_file_i00= beam_file.replace('.fits', '_I000.fits')
```

Comment:
Why these changes? `black` will revert them back immediately and they clutter the review. It would be helpful to format the source before committing.